eval/src/tests/tensor/tensor_slime_serialization/tensor_slime_serialization_test.cpp

```diff
@@ -63,8 +63,8 @@ vespalib::string twoCellsJson[3] =
     "] }",
     "{ dimensions: [ 'x', 'y' ],"
     " cells: ["
-    "{ address: { x:'1'}, value: 3.0 },"
-    "{ address: { y:'3'}, value: 4.0 }"
+    "{ address: { y:'3'}, value: 4.0 },"
```

Comment:
prefer parsing json to slime and compare without order-sensitivity

Reply:
How, as far as I can tell it is parsed to slime now. And then it compares the 2 slime objects for equality. Which fails.

Reply:
my bad. since cells is a list and not a map, the order IS relevant :-p